components/butterfly/src/server/mod.rs

```diff
@@ -1231,6 +1252,96 @@ impl<'a> Serialize for ServerProxy<'a> {
 
 #[cfg(test)]
 mod tests {
+    use super::*;
+    use habitat_core::service::ServiceGroup;
+
+    impl Default for Service {
```

Comment:
Rather than implementing `Default` in a test module, could this just be a helper function that takes a `member_id` and `service_group`?

Reply:
Good call; that's a bit simpler and considerably more evocative